app/src/androidTest/java/ch/epfl/balelecbud/utility/json/JsonResourceReaderTest.java

```diff
@@ -0,0 +1,46 @@
+package ch.epfl.balelecbud.utility.json;
+
+import org.junit.Test;
+
+import java.util.List;
+
+import ch.epfl.balelecbud.R;
+import ch.epfl.balelecbud.model.HelpPage;
+
+import static org.testng.AssertJUnit.assertEquals;
```

Comment:
Is there a reason why you are using `org.testng.AssertJUnit.assertEquals` instead of `org.junit.Assert.assertEquals`